reply.lua

```diff
@@ -87,23 +96,25 @@ function net.tcphdr(packet)
 	}, offset
 end
 
-function net.pseudohdr(ip, tcp)
-	local pseudo = data.new(12 + tcp.doff * 4)
+function net.pseudohdr(ip, tcp, payload)
+	local payload_len = payload and #payload or 0
+	local pseudo = data.new(12 + tcp.doff * 4 + payload_len)
```

Comment:
What's `12` here? Can we put this in a local var? Also, I would store `doff * 4 + (#payload or 0)` above instead of `payload_len` because it is actually what's used here and below..